docker-compose.microservices.example.yml

```diff
@@ -0,0 +1,117 @@
+    build:
+      context: .
+      dockerfile: Dockerfile.microservices
+    environment:
+      # LOCAL ONLY!
+      - LOG_NEW_LINES=1   # Logs new lines instead of <\n>
+      # DATA CLEARING
+      - CLEAR_DATA_KEY=123                                                              # Change this if you want to delete filesystem data on startup.
+      # Container
+      - STARTUP_WAIT_TIMEOUT=30                                                         # The amount of times to attempt to connect to L2 geth before failing (set to -1 if not to wait for L2 geth).
+      # Logging
+      - DEBUG=info*,error*,warn*,debug*                                                 # The comma-separated logging patterns to match (common options are `error*`, `info*`, `warn*`, and `debug*`)
+      # Postgres
+      - POSTGRES_HOST=postgres                                                          # (Required) The host DNS entry / IP for the postgres DB
+      - POSTGRES_PORT=5432                                                              # (Required) Should almost always be 5432
+      - POSTGRES_USER=test                                                              # (Required) The user to use to connect to the db
+      - POSTGRES_PASSWORD=test                                                          # (Required) The password to use to connect to the db
+      - POSTGRES_DATABASE=rollup                                                        # (Required) The database name to connect to (should be `rollup`)
+      - POSTGRES_CONNECTION_POOL_SIZE                                                   # The connection pool size for postgres (defaults to 20)
+      - POSTGRES_USE_SSL                                                                # Set to anything to indicate that SSL should be used in the connection
+      # L1 Node
+      - L1_NODE_INFURA_NETWORK=rinkeby                                                  # The Infura network for the connection to the L1 node
+      - L1_NODE_INFURA_PROJECT_ID=91981cfffb524ceca0c2e5b18905f9f5                      # The Infura project ID for the connection to the L1 node
+      - L1_NODE_WEB3_URL                                                                # The URL of the L1 node
+      - FINALITY_DELAY_IN_BLOCKS=1                                                      # The number of block confirmations required to consider a transaction final on L1
+      # L2 Node
+      - L2_NODE_WEB3_URL=http://geth_l2:8545                                            # The URL of the L2 node
+      # L1 Submitters
+      - L1_SEQUENCER_PRIVATE_KEY=<TODO: Add this>                                                   # (Required) The private key to use to submit Sequencer Transaction Batches
+      - L1_STATE_ROOT_PRIVATE_KEY=<TODO: Add this>                                                  # (Required) The private key to use to submit State Root Batches (will cause nonce issues if the same as the above key)
```

Comment:
```suggestion
      - L1_STATE_ROOT_SUBMITTER_PRIVATE_KEY=<TODO: Add this>                                                  # (Required) The private key to use to submit State Root Batches (will cause nonce issues if the same as the above key)
```

Reply:
Looks good. Not committing it here, though since all references to it need to be updated as well.